my-project/src/Migrations/Version20180608100535.php

```diff
@@ -1,28 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
@mSobonVirtua ta migracja jest niepotrzebna?

Reply:
Nie wiem

Reply:
Skoro dodaje tabele produktu i nie ma tego nigdzie indziej zadeklarowanego to trzeba ja przywrocic